lookyloo/modules/uwhois.py

```diff
@@ -87,7 +90,13 @@ def whois(self, query: str, contact_email_only: bool=False) -> str | list[str]:
                 if not data:
                     break
                 bytes_whois += data
+        #if an abuse-c-Object is found in the whois entry, the result of its lookup will be returned
+        abuse_c = re.search(rb'abuse-c:\s+(.*)\s', bytes_whois)
+        if abuse_c is not None:
+            return self.whois(abuse_c.group(1).decode(), contact_email_only)
```

Comment:
We probably want to concatenate both requests an return them together, *unless* `contact_email_only` is `True` in which case the emails from the `abuse_c` response take priority (if they exist).